bin/npg_delete_dr_runs

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use FindBin qw($Bin);
+use lib ( -d "$Bin/../lib/perl5" ? "$Bin/../lib/perl5" : "$Bin/../lib" );
+use Carp;
+use English qw(-no_match_vars);
+use Getopt::Long;
+use Pod::Usage;
+
+use npg_tracking::Schema;
+
+our $VERSION = '0';
+
+my $help;
+my $files;
+my $folders;
+my $no_dry_run;
+
+sub send_commands {
+    my ($dr_server, $cmd) = @_;
+    open(BASH, "| ssh $dr_server bash -e") or die("Can't run ssh: $!");
+    print BASH $cmd;
+    close(BASH);
+}
+
+GetOptions ('help|?'   => \$help,
+            'no-dry-run'=> \$no_dry_run,
+            'files'    => \$files,
+            'folders'  => \$folders)
+  or croak("Error in command line arguments\n");
+
+if ($files and $folders) {
+    print STDERR "You can't have --files AND --folders. Choose one.\n";
+    pod2usage(1);
+}
+
+
+pod2usage(0) if $help;
+
+my $rs=npg_tracking::Schema->connect()->resultset(q(RunStatusDict))
+    ->search({description=>[q[run cancelled],q[data discarded],q[qc complete]]})
+    ->related_resultset(q(run_statuses))->search({q(run_statuses.iscurrent)=>1})
+    ->related_resultset("run")->search({q(run.folder_path_glob)=>{LIKE=>q(%/esa-%)}});
+
+my $cmd = '';
+while (my $run=$rs->next()) {
+    my $status = $run->current_run_status_description || "";
+    my $tag = $run->is_tag_set("staging");
+    my $run_folder = $run->folder_name || "";
+    if ($run_folder and !$tag ) {
+        my $prefix = substr($run_folder,0,4);
+        my $dname = "/exports/$prefix/$run_folder";
+        if ($files and !$folders) {
+            $cmd .= "if [ -d $dname ]; then rm -f $dname/Data/Intensities/BaseCalls/L\*/C\*/\*.cbcl; fi\n";
+        }
+        if ($folders and !$files) {
+            $cmd .= "if [ -d $dname ]; then rm -r $dname; fi\n";
+        }
+        if (!$files and !$folders) {
+            $cmd .= "if [ -d $dname ]; then ls -ld $dname; fi\n";
+        }
+    }
+}
+chop $cmd;
+if ($no_dry_run) {
+    send_commands('ukbb\@seq-stor-01', $cmd);
```

Comment:
Uncomfortable with hostnames hardcoded (in this repo) - if staying in `npg_tracking` these should be able to be set from command line options (but they could be defaults - but include as fqdn if so).

Reply:
Simplifying things is good. I like simple.